frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -502,6 +502,15 @@ export const deletePublicLink = createAction(DELETE_PUBLIC_LINK, async ({ id })
     return { id };
 });
 
+import * as Urls from "metabase/lib/urls";
+import { push } from "react-router-redux";
+
+const CHANGE_CARD_AND_RUN = "metabase/database/CHANGE_CARD_AND_RUN";
+export const onChangeCardAndRun = createThunkAction(CHANGE_CARD_AND_RUN, card =>
```

Comment:
Could it be explained here that what are the use cases for this action? It's not immediately obvious to me